packages/test-utils/package.json

```diff
@@ -41,7 +41,8 @@
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
     "react-router": "^6.0.0-alpha.5",
-    "react-router-dom": "^6.0.0-alpha.5"
+    "react-router-dom": "^6.0.0-alpha.5",
+    "zen-observable": "^0.8.15"
```

Comment:
Why this specific Observable library?

Reply:
Oh. Nevermind. I can see we already use it in other places.

Reply:
Yep, it's a very lightweight one that pretty much just implements the core spec. It's also a dependency of some of our other dependencies so it comes "for free" 😁